docs/development-setup/deploying-a-runtime.md

```diff
@@ -333,9 +333,9 @@ oasis-node \
   --runtime.provisioner unconfined \
   --runtime.supported $RUNTIME_ID \
   --runtime.paths $RUNTIME_ID=$RUNTIME_BINARY \
-  --consensus.tendermint.debug.addr_book_lenient \
-  --consensus.tendermint.debug.allow_duplicate_ip \
-  --consensus.tendermint.p2p.seed $SEED_NODE_ADDRESS \
+  --consensus.cometbft.debug.addr_book_lenient \
+  --consensus.cometbft.debug.allow_duplicate_ip \
+  --consensus.cometbft.p2p.seed $SEED_NODE_ADDRESS \
```

Comment:
None of these examples probably work given the recent config changes?

Reply:
Yes, these should be updated in a separate PR.